quill-cassandra/src/main/scala/io/getquill/context/cassandra/encoding/Encodings.scala

```diff
@@ -16,4 +16,9 @@ trait Encodings extends CassandraMapperConversions with CassandraTypes {
 
   implicit val encodeJava8Instant: MappedEncoding[Instant, Date] = MappedEncoding(Date.from)
   implicit val decodeJava8Instant: MappedEncoding[Date, Instant] = MappedEncoding(_.toInstant)
+
+  implicit val encodeJava8ZonedDateTime: MappedEncoding[ZonedDateTime, Date] = MappedEncoding(zdt =>
+    Date.from(zdt.toInstant))
+  implicit val decodeJava8ZonedDateTime: MappedEncoding[Date, ZonedDateTime] = MappedEncoding(d =>
+    ZonedDateTime.ofInstant(d.toInstant, ZoneId.systemDefault))
```

Comment:
Could you make the zone configurable?

Reply:
I am not sure what's the best way to do it. Can you suggest me something?

Reply:
@mentegy done. Thanks!